src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -502,6 +502,11 @@ private function addServiceMethodCalls($id, Definition $definition, $variableNam
     private function addServiceOverriddenMethods($id, Definition $definition)
     {
         $class = $this->container->getReflectionClass($definition->getClass());
+
+        if (!$class) {
+            throw new RuntimeException(sprintf('Unable to configure service "%s": class could not be reflected.', $id));
```

Comment:
`[...] class "%s" not found.`?
can you review the other uses of getReflectionClass in this class and also in ContainerBuiler please?